docs/data/joy/customization/dark-mode/DarkModeByDefault.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { CssVarsProvider } from '@mui/joy/styles';
+import Sheet from '@mui/joy/Sheet';
+import Chip from '@mui/joy/Chip';
+import Typography from '@mui/joy/Typography';
+
+export default function DarkModeByDefault() {
+  return (
+    <CssVarsProvider
+      defaultMode="dark"
+      // the props below are specific to this demo, you might not need them in your app.
+      colorSchemeSelector="#demo_dark-mode-by-default" // the selector to apply CSS theme variables stylesheet.
```

Comment:
If we care about the debugging use case here, I think that it would be better to have an intermediary `<div>` to host the styles. When you open the dev tools to find what are the styles of `<Sheet>`, we get flooded with CSS variables properties:

<img width="332" alt="Screenshot 2022-11-07 at 02 19 56" src="https://user-images.githubusercontent.com/3165635/200207613-05d8a3aa-931b-42f9-8bc2-92666b70e86e.png">

Reply:
Done!